ebl/src/main/java/org/dcsa/conformance/standards/ebl/checks/EblDatasets.java

```diff
@@ -63,4 +53,28 @@ public class EblDatasets {
     "CUS"
   );
 
+  public static final KeywordDataset DOCUMENTATION_PARTY_CODE_LIST_PROVIDER_CODES = KeywordDataset.staticDataset(
+    "WAVE",
+    "CARX",
+    "ESSD",
+    "IDT",
+    "BOLE",
+    "EDOX",
+    "IQAX",
+    "SECR",
+    "TRGO",
+    "ETEU",
+    "TRAC",
+    "BRIT",
+    "GSBN",
+    "WISE",
+    "GLEIF",
+    "W3C",
+    "DNB",
+    "FMC",
+    "DCSA",
+    "EU",
```

Comment:
As per the new snapshot the EU option has been removed from this list.

Reply:
Thanks, should be removed now. :)